src/server.js

```diff
@@ -0,0 +1,38 @@
+const Hapi = require("hapi")
+
+const server = (cluster) => {
+  const server = new Hapi.Server()
+
+  server.connection({ port: process.env.PORT || 8080 })
+
+  server.route({
+    method: "GET",
+    path: "/",
+    handler: (request, reply) => {
+      const response = reply('Server running')
+      response.type('text/plain')
+      response.statusCode = 200
+    }
+  })
+
+  server.route({
+    method: "GET",
```

Comment:
I went with `GET` because the main web app's callback uses a GET request, but I think it should be `DELETE`. Will change.